packages/EasyDoctrine/src/Dispatchers/DeferredEntityEventDispatcher.php

```diff
@@ -48,6 +53,13 @@ public function __construct(EventDispatcherInterface $eventDispatcher)
         $this->eventDispatcher = $eventDispatcher;
     }
 
+    public function addToDisableList(array $objectClasses): void
```

Comment:
Such a method creates implicit dependency. The enable/disable calls become not self-sufficient. I mean in some cases to enable or disable events we have to make additional calls to make the logic work as expected. It's like pre-configuration.
I believe we shouldn't do pre-configuration, and methods calls should be self-sufficient.

Reply:
Pls have a look again